doorman/assets.py

```diff
@@ -7,6 +7,7 @@
            filters='less',
     ),
     'libs/bootstrap-tagsinput/dist/bootstrap-tagsinput.css',
+    'libs/jQuery-QueryBuilder/dist/css/query-builder.default.css',
```

Comment:
```
.query-builder .rule-value-container select, .query-builder .rule-value-container input[type=text], .query-builder .rule-value-container input[type=number] {
  padding: 1px;
}
```

![image](https://cloud.githubusercontent.com/assets/359029/15575540/646a63d8-2321-11e6-9360-1e3d6764ebd9.png)

is not enough padding. I tried to override it, (setting `padding: 8px`) in style.css, but it keeps getting overridden.

![image](https://cloud.githubusercontent.com/assets/359029/15575400/a95c4ff2-2320-11e6-8231-b9f03707d5ee.png)

And this would be better with 2px margin, not 5px.

```
.query-builder .rule-container .rule-filter-container,
.query-builder .rule-container .rule-operator-container,
.query-builder .rule-container .rule-value-container, .query-builder .error-container, .query-builder .drag-handle {
  display: inline-block;
  margin: 0 5px 0 0;
  vertical-align: middle;
}
```

![image](https://cloud.githubusercontent.com/assets/359029/15575607/ae703458-2321-11e6-936b-19da3f29820f.png)

Not a big deal, but would be nice to spruce it up a bit.